SnakeGame/tests.py

```diff
@@ -348,6 +349,24 @@ def test_contians_invalid(self):
         coord = 3
         self.assertFalse(coord in self.snake)
 
+class GameViewtest(TestCase):
+
+    def setUp(self):
+        self.grid = [
+            [CellContent.SnakeBody, CellContent.SnakeBody, CellContent.SnakeBody, CellContent.SnakeBody],
+            [CellContent.Empty, CellContent.SnakeBody, CellContent.Empty, CellContent.Empty],
+            [CellContent.Empty, CellContent.Empty, CellContent.SnakeBody, CellContent.Empty],
+            [CellContent.SnakeBody, CellContent.Empty, CellContent.Empty, CellContent.SnakeHead]
+        ]
+
+    def test_viewIsRenderedWithoutException(self):
```

Comment:
as we discussed, I'm not sure if this will work in github's containers, seeing as there is no screen. We'll try this anyways once we merge the cotinuous-integration pull request, and see where we go from there.